src/bulk/common.ts

```diff
@@ -161,6 +161,8 @@ export class Batch {
 export class BulkWriteResult {
   result: BulkResult;
 
+  /** Indicates whether this write result was acknowledged. If not, then all other members of this result will be undefined */
+  // acknowledged: Boolean;
```

Comment:
Is this supposed to be commented out?

Reply:
yeah this is deferred to forthcoming bulk typing work